### PR TITLE
Caresa Huddleston

### PR DESCRIPTION
**Pull request by caresa**

### CODE REVIEW

File: library.rb [hunk 1]
```diff
@@ -1,37 +1,115 @@
...
 
   def check_in_book(book)
+
+    if book && book.status =="checked_out"
+       book.status = "available"
+
+    end
   end
 
   def available_books
```
Comment (truffaut) [inline library.rb:92]: this method works, but you could write it in a simpler way using the `select` method.

You could replace the entire contents of the method with:

``` ruby
@books.select do |book|
  book.status == "available"
end
```

Comment (caresa) [inline library.rb:92]: Thanks!!

On Wed, Apr 30, 2014 at 12:02 PM, Michael Ornellas <notifications@github.com

File: library.rb [hunk 2]
```diff
@@ -1,37 +1,115 @@
...
+
+    @books.each do |book|
+      if book.status == "available"
+          avail_books << book
+        end
+      end
+       avail_books
   end
 
   def borrowed_books
```
Comment (truffaut) [inline library.rb:104]: same thing for this method as the one above

**Comment by truffaut** [comment]
Good work! Remember to commit often–its much, much easier to undo small changes than bigger ones. :thumbsup:

**Pull request closed without merging**